src/objects.h

```diff
@@ -1354,6 +1384,10 @@ class Instance : public HeapObject {
     _at_put(_offset_from(index), value);
   }
 
+  INLINE Object** root_at(int index) {
```

Comment:
const

Reply:
No we can modify the object via this call.